src/main/Extensions/Base64Conversion/Types.ts

```diff
@@ -0,0 +1,9 @@
+export type InvocationArgument = {
+    invocationString: string;
+    encodingType: EncodingType;
+};
```

Comment:
I find the names "invocationString" and "encodingType" not very good. "decode" is not an encoding type. To me "decode" and "encode" are more like "actions" or something similar. And the "invocationString" is maybe a "payload"? What do you think about this?


```typescript
export type InvocationArgument = {
    action: "encode" | "decode";
    payload: string;
}
```

Reply:
I agree, naming is hard and I wasn't paying enough attention when I wrote this type.